### DISCUSSION TITLE
Processor device amd gpu not found

### DISCUSSION THREAD

**Discussion by Wanniox** [nagadomi/nunif#162] [Q&A] [answered]
I have a 6800 xt. I can only select cpu as device in processor.

**Comment by nagadomi** [comment, marked as answer]
Currently this repository does not support AMD GPU.

On Windows, PyTorch for ROCm is not yet supported. https://github.com/pytorch/pytorch/issues/106608 https://github.com/ROCm/ROCm/issues/2915
On Linux it may work if you install PyTorch for ROCm, but I have never tried it because I don't have an AMD GPU.

**Comment by beosliege** [comment]
Please refer to this thread:
https://github.com/nagadomi/nunif/discussions/166

I do not see the 6800xt on AMD's supported list of GPUs, but as shown, iw3 is now functional on Windows (using WSL) for specific Radeon Instinct / Pro / Non-Pro GPUs.
